### ISSUE TITLE
Fabric Easy Elevators Mixin config conflicting with itself on start (1.0/1.1)

### ISSUE THREAD

**Issue by painedanomaly** [ricksouth/serilum-mc-mods#2577]
## **Information**
Minecraft version: 1.20.1
Modloader: Fabric
Fabric loader version: 0.16.0
Environment: Singleplayer / Multiplayer

Mod name: Easy Elevators
Mod version: 1.0 / 1.1

## **Description**
I don't know too much about the actual code of mods and such, but it seems like Easy Elevators is somehow being prevented from loading by itself. It says there's "Non-unique Mixin config name easyelevators_fabric.mixins.json used by the mods easyelevators and easyelevators." It does this with both versions of the mod installed, I even checked in the mods folder to make sure Curseforge didn't accidentally download the Forge version as well, but it was only the one Fabric version every time.

I figured I'd report this sooner rather than later, because I've played with this mod on Forge and it's honestly great having a modded elevator that keeps within vanilla Minecraft. The only problem with this mod came when I tried making a Fabric modpack, and it just keeps knocking itself out every time I load the game. It works fine without it, sadly, but I don't want that to be the case, you know?

## **Crash report**
It didn't give me a crash report, but I do have the log [here](https://gist.github.com/painedanomaly/03c60a1bdab6e134806cfe5365a7ecce).
I also have a picture of the Fabric loader crash window [here](https://imgur.com/a/vk3VCt2), if that helps any at all.

**Comment by ricksouth** [comment]
It sounds like you had the mod loaded twice. Did you install both version 1.1 and 1.0 at the same time?

**Comment by painedanomaly** [comment]
No, I made sure that I only had 1 version installed. I manually checked every time, but for some reason it loads itself twice on startup no matter which version. Once I delete the mod, it launched perfectly fine, but having either one installed brings up that same window.

Edit: I made a brand new modpack only running Sodium, JEI, Easy Elevators and Collective, and the same error message still appears.
